tree/20_cfe_basics/ncf_lib.cf

```diff
@@ -901,7 +901,7 @@ bundle agent ncf_services(service, action)
 
     # init.d
     # WARN: Some actions may not be supported by the init script
-    pass1.!is_boot_action.!systemctl_utility_present.!is_upstart_service.!svcadm_utility_present.!service_utility_present.!startsrc_utility_present.init_d_directory_present::
+    pass1.!is_boot_action.!systemctl_utility_present.!is_upstart_service.!svcadm_utility_present.!service_utility_present.!startsrc_utility_present.is_init_service::
```

Comment:
this condition is invalid: it may be a systemctl_utiliy_present, but with a init script
it should be (!systemctl_utility_present|is_init_service) also